admin/react/assets/src/dashboard-app/pages/fonts/preview/GooglePreviewItem.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { __ } from "@wordpress/i18n";
+import { useSelector } from 'react-redux';
+import GFontVariation from './GFontVariation';
+
+const GooglePreviewItem = () => {
+	const googleFontState = useSelector( ( state ) => state.googleFont );
+
+	if ( undefined === googleFontState ) {
+		return;
+	}
+
+	const googleFont = googleFontState.font_name;
+	const gFontData = bsf_custom_fonts_admin.googleFonts[googleFont];
+	if ( undefined === gFontData ) {
+		return;
+	}
+	const variations = gFontData[0] ? gFontData[0] : [];
+
+	const getGoogleFontLink = (font, version) => {
+		const fontName = font.replace( / /g, "+" );
+		return `https://fonts.googleapis.com/css?family=${fontName}&ver=${version+1}`;
```

Comment:
Maybe keep this in a global variable so that it can be reused 

https://fonts.googleapis.com/css?family @imnavanath

Reply:
DONE localized it and reused 👍